dotCMS/src/main/java/com/dotmarketing/util/ExportStarterUtil.java

```diff
@@ -0,0 +1,305 @@
+                        _dh.setQuery("from " + clazz.getName() + " order by parent, child, relation_type");
+                    }
+                    else if(MultiTree.class.equals(clazz)){
+                        _dh.setQuery("from " + clazz.getName() + " order by parent1, parent2, child, relation_type");
+                    }
+                    else if(TagInode.class.equals(clazz)){
+                        _dh.setQuery("from " + clazz.getName() + " order by inode, tag_id");
+                    }
+                    else if(Tag.class.equals(clazz)){
+                        _dh.setQuery("from " + clazz.getName() + " order by tag_id, tagname");
+                    }
+                    else if(CalendarReminder.class.equals(clazz)){
+                        _dh.setQuery("from " + clazz.getName() + " order by user_id, event_id, send_date");
+                    }
+                    else if(Identifier.class.equals(clazz)){
+                        dc = new DotConnect();
+                        dc.setSQL("select * from identifier order by parent_path, id")
+                                .setStartRow(i).setMaxRows(step);
+                    }
+                    else if (Language.class.equals(clazz)) {
+                        dc = new DotConnect();
+                        dc.setSQL("SELECT * FROM language order by id")
+                                .setStartRow(i).setMaxRows(step);
+                    }
+                    else if (Relationship.class.equals(clazz)) {
+                        dc = new DotConnect();
+                        dc.setSQL("SELECT * FROM relationship order by inode")
+                                .setStartRow(i).setMaxRows(step);
+                    }
+                    else {
+                        _dh.setQuery("from " + clazz.getName() + " order by 1");
+                    }
+
+                    if(Identifier.class.equals(clazz)){
+                        _list = TransformerLocator
+                                .createIdentifierTransformer(dc.loadObjectResults()).asList();
+                    } else if (Language.class.equals(clazz)) {
+                        _list = TransformerLocator
+                                .createLanguageTransformer(dc.loadObjectResults()).asList();
+                    } else if (Relationship.class.equals(clazz)) {
+                        _list = TransformerLocator
+                                .createRelationshipTransformer(dc.loadObjectResults()).asList();
+                    } else {
+                        _list = _dh.list();
+                    }
+                    if(_list==null ||_list.isEmpty()) {
+                        break;
+                    }
+                    if(_list.get(0) instanceof Comparable){
+                        try {
+                        java.util.Collections.sort(_list);
+                        }
+                        catch(java.lang.IllegalArgumentException e) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A catch statement that catches an exception only to rethrow it should be avoided.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5632525)